src/test/java/org/example/c01operators/calculate/DaysToYearsWeeksAndDaysTest.java

```diff
@@ -5,29 +5,31 @@
 import static org.junit.jupiter.api.Assertions.*;
 
  /*
+ Write a code in Java to transform numbers of days in years, weeks and days.
+
  I create an arrays with length=3. First position is for number of years
         second for number of weeks and last one for number of days
 */
 
 public class DaysToYearsWeeksAndDaysTest {
     @Test
-    public void tenLeapYear() {
+    public void given3660AsNumberOfDaysShouldBeTenLeapYear() {
```

Comment:
This test is not correct. In 10 years there are 2 leap years.

Reply:
done

Reply:
where?